### ISSUE TITLE
having chrome dev tools open causes strange behavior for transition

### ISSUE THREAD

**Issue by 719media** [justintaddei/v-shared-element#3]
If you open up chrome dev tools, there will be strange transition behavior. To me, this is a sign of a race condition somewhere that is not being taken into account (not sure)?

This caused me quite a bit of confusion until I realized the culprit was having devtools open.

![transition-problem](https://user-images.githubusercontent.com/5942229/83564056-f774c800-a4d0-11ea-80c4-fcaadd297d78.gif)

The sourcecode for this issue is here:
https://codesandbox.io/s/sharedelementtransition-consolelog-problem-ydi4k

**Comment by justintaddei** [comment]
Which version of Chrome are you running? I'm not able to recreate the issue.

**Comment by justintaddei** [comment]
Wait. I figured it out. You have your cache disabled on the network panel. The image's `height` is not known at the time of the transition. I've never used `lazysizes` so I don't really know how it works, but it seems to be the root of the problem.

**Comment by 719media** [comment]
Sorry, lazysizes was not a part of this example. I've removed it from the source example mentioned above.
Good catch on the disable cache. I did not stop to think that the image size would need to be known prior to animation, but it makes sense. This reproduction was a simplification of another problem I was having, and I thought this was the issue. Going to close this issue until I can get a better reproduction.
